### ISSUE TITLE
FEATURE: There's raw domain output, but no raw IP output.

### ISSUE THREAD

**Issue by T145** [funilrys/PyFunceble#268]
## Description

It would be nice if there were a way to get directly IPs rather than from just the hosts file output.

## Possible Solution

On that note, it would also be great to at least leave `hosts_ip` empty and then output raw lists under `hosts`.
This would probably be the better option as opposed to having separate "domains" and "ips" outputs.
Then the present "domains" output becomes obsolete.

## Considered Alternative

Generating an "ips" output, but it's lazy.

## Additional context

It may also be worth considering how to handle IPv6 addresses.

**Comment by spirillen** [comment]
A good idea. I defiantly second that as I handle IP my self both for firewall like nftables and RPZ zones, haven a clear & clean (IP|IP/CIDR) would be a nice feature.

**Comment by funilrys** [comment]
By the way, not that this has been fixed, I wanted to explain the reason why `ips` weren't "first-class citizens" among the `plain` output format. 

Back when I developed [funceble](https://github.com/funilrys/funceble) - for my own usage, funceble was a tool that was highly hosts-file oriented. As IPs are useless into hosts files, I used that method to filter out IPs.

Now that this tool is used by many others, it was time to fix this. Sorry for the waiting time @T145, the last 2-3 years have been hard - privately. 

Stay safe and healthy.
